### ISSUE TITLE
"Every day at 9am" stopped working in v2.0.0rc

### ISSUE THREAD

**Issue by bjonord** [yb66/tickle#21]
Not sure if this is being worked on or if the incantation is going to be dropped?

```
> Tickle.parse('every day at 9am')
Traceback (most recent call last):
        1: from (irb):12
ArgumentError (argument out of range)
```

This works as before: 

```
> Tickle.parse('every tuesday')
=> {:next=>2018-11-06 12:00:00 +0000, :expression=>"every tuesday", :starting=>2018-10-31 09:28:52 +0000, :until=>nil}
```

Let me know if I can lend a hand.

Cheers

**Comment by yb66** [comment]
Hi @bjonord 

Development has stalled on this, because the codebase is a bit too messy to do anything in a short time frame and other stuff gets in the way (life, plans… and bugs!) and Chronic, which Tickle depends on, keeps threatening a new release which also puts me off.

As to your problem, it does seem that phrasing dropped out of the code at some point, my apologies. I'll add a spec for it in `spec/tickle_spec.rb` and if you want to have a crack at it, by all means. I'll give it a look too, it'll be Friday afternoon though.

Regards,
iain

**Comment by yb66** [comment]
I've added a branch with the failing test here https://github.com/yb66/tickle/blob/issue-21-every-day-at-9am/spec/tickle_spec.rb#L443

**Comment by bjonord** [comment]
> Chronic, which Tickle depends on, keeps threatening a new release which also puts me off.

I noticed that too. Very unfortunate. Have you gotten an indication of how breaking the new release will be?

>if you want to have a crack at it, by all means.

Not sure when I'll get a chance to have look, possibly tomorrow afternoon as well.

**Comment by yb66** [comment]
> Have you gotten an indication of how breaking the new release will be?

Not at all. It was on a private branch for a while, and may be public now but it's been spoken of for so long that I decided to wait.

My plans for change, roughly, are to change Tickle's API, possibly into an (lazy) enumerator with a more standard ISO time response (I'd need to re-read my notes). The guts can change when Chronic changes.

iain

**Comment by bjonord** [comment]
Started, but got pulled away on other stuff.

One thing I noted, the branch above is branched from RC1 instead of RC2?

**Comment by yb66** [comment]
Well spotted, for some reason I didn't have rc2. Weird. Anyway, I've fixed that. I also was sidetracked, I'll have a go at it tomorrow (I'm on Japan time).

iain

**Comment by yb66** [comment]
Hi @bjonord 

After wading through the code it seems Chronic was handling that bit of the parsing. I'm going to see if I can hack something in that works instead, it may or may not work. The Chronic rewrite doesn't handle it either. I'll let you know if I can get it to work.

iain
